docs/operator-docs/serve-dashboard-from-cloudfront.md

```diff
@@ -0,0 +1,26 @@
+---
+id: serve-dashboard-from-cloudfront
+title: Serve Dashboard with CloudFront
+hide_title: true
+---
+
+# Serve Cumulus Dashboard behind CloudFront
```

Comment:
I think NASD/NGAP/Platform specific documentation should go into the onboarding documentation in the wiki, especially since we already link back to the wiki in a few places here. As part of the epic next PI I am going to have us move Platform specific docs to the internal wiki.